README.md

```diff
@@ -1,44 +1,54 @@
-CBE Thermal Comfort Tool
-========================
+# CBE Thermal Comfort Tool
 
-A web interface for comfort model calculations and visualizations according to ASHRAE Standard-55. [Live deployment of the tool](http://smap.cbe.berkeley.edu/comforttool).
+A web interface for comfort model calculations and visualizations according to ASHRAE Standard-55 and EN Standard 15251. 
 
-Installation
-------------
+[Live deployment of the tool](http://comfort.cbe.berkeley.edu/).
 
-This guide is for Mac OSX or Linux. For Windows, you can follow the instructions below, substituting the appropriate windows commands found in the [Flask installation instructions](http://flask.pocoo.org/docs/0.10/installation/).
+## Getting Started
 
-The installation is easiest with [pip](http://pip.readthedocs.org/en/latest/installing.html), a tool for managing python packages.
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
 
-Once you have pip, start by installing virutalenv:
+## Prerequisites
 
-`$ sudo pip install virtualenv`
+Python 3 installed on your machine.
 
-Next, check out from the repo and create a virtualenv:
+If you’re using Python 2, see [Install virtualenv](https://flask.palletsprojects.com/en/1.1.x/installation/#install-install-virtualenv) first.
```

Comment:
Does Python2 still work?

Reply:
I have not tested that, I guess so but I will remove that comment.